apps/root/src/pages/dca/positions/components/positions-list/position-card/index.tsx

```diff
@@ -0,0 +1,535 @@
+                    <ContainerBox flexDirection="column" gap={1}>
+                      <StyledBodySmallTypography>
+                        <FormattedMessage description="current remaining" defaultMessage="Remaining:" />
+                      </StyledBodySmallTypography>
+                      <ContainerBox gap={1} alignItems="center">
+                        <TokenIcon size={5} token={from} />
+                        <StyledBodySmallTypography fontWeight={700}>
+                          {formatCurrencyAmount(totalRemainingLiquidity, position.from, 4)}
+                        </StyledBodySmallTypography>
+                        {showFromPrice && (
+                          <>
+                            <StyledBodySmallTypography fontWeight={700}>·</StyledBodySmallTypography>
+                            <Typography variant="bodySmall">${fromPrice.toFixed(2)}</Typography>
+                          </>
+                        )}
+                      </ContainerBox>
+                    </ContainerBox>
+                    {remainingSwaps > 0n && (
+                      <ContainerBox gap={1}>
+                        <StyledBodySmallTypography>
+                          <FormattedMessage description="positionDetailsNextSwapTitle" defaultMessage="Next swap:" />
+                        </StyledBodySmallTypography>
+                        {DateTime.now().toSeconds() < DateTime.fromSeconds(position.nextSwapAvailableAt).toSeconds() ? (
+                          <StyledBodySmallTypography fontWeight={700}>
+                            {DateTime.fromSeconds(position.nextSwapAvailableAt).toRelative()}
+                          </StyledBodySmallTypography>
+                        ) : (
+                          <StyledBodySmallTypography fontWeight={700}>
+                            <FormattedMessage
+                              description="positionDetailsNextSwapInProgress"
+                              defaultMessage="in progress"
+                            />
+                          </StyledBodySmallTypography>
+                        )}
+                      </ContainerBox>
+                    )}
+                  </ContainerBox>
+                }
+              >
+                <div>
+                  <PositionProgressBar
+                    value={totalSwaps === 0n ? 0 : Number((100n * (totalSwaps - remainingSwaps)) / totalSwaps)}
+                  />
+                </div>
+              </Tooltip>
+            </ContainerBox>
+            {isTestnet && (
+              <ContainerBox alignItems="flex-start">
+                <Chip
+                  label={<FormattedMessage description="testnet" defaultMessage="Testnet" />}
+                  size="small"
+                  color="warning"
+                />
+              </ContainerBox>
+            )}
+            {((position.from.symbol === 'CRV' && foundYieldFrom) || (position.to.symbol === 'CRV' && foundYieldTo)) && (
```

Comment:
Can you move all of this warning messages into its own component? so its easier to modify in the future

Reply:
Absolutely!